src/test/java/com/epam/tc/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,156 @@
+
+        List<String> indexPageImages = List.of("icons-benefit", "icon-custom", "icon-multi", "icon-base");
+
+        for (String indexPageClassName : indexPageImages) {
+            webElement = webDriver.findElement(By.className(indexPageClassName));
+            boolean indexPagePicture = webElement.isDisplayed();
+            assertThat(indexPagePicture)
+                .as("Index page some picture")
+                .isTrue();
+        }
+
+        //        7. Assert that there are 4 texts on the Index Page under icons and they have proper text
+
+        List<WebElement> indexPageFourTexts = webDriver.findElements(By.className("benefit-txt"));
+        assertThat(indexPageFourTexts.get(0).getText()).isEqualTo("To include good practices\n"
+            + "and ideas from successful\n"
+            + "EPAM project");
+        assertThat(indexPageFourTexts.get(1).getText()).isEqualTo("To be flexible and\n"
+            + "customizable");
+        assertThat(indexPageFourTexts.get(2).getText()).isEqualTo("To be multiplatform");
+        assertThat(indexPageFourTexts.get(3).getText()).isEqualTo("Already have good base\n"
+            + "(about 20 internal and\n"
+            + "some external projects),\n"
+            + "wish to get more…");
+
+        //        8. Assert that there is the iframe with “Frame Button” exist
+
+        webElement = webDriver.findElement(By.id("frame"));
+        boolean iframeExists = webElement.isEnabled();
+        assertThat(iframeExists)
+            .as("Iframe exists")
+            .isTrue();
+
+        //        9. Switch to the iframe and check that there is “Frame Button” in the iframe
+
+        webDriver.switchTo().frame("frame");
+
+        webElement = webDriver.findElement(By.xpath("//*[@id='frame-button']"));
+        boolean frameButtonExists = webElement.isEnabled();
+        assertThat(frameButtonExists)
+            .as("Button frame exists")
+            .isTrue();
+
+        //        10. Switch to original window back
+
+        webDriver.switchTo().defaultContent();
+
+        //        11. Assert that there are 5 items in the Left Section are displayed and they have proper text
+
+        String[][] leftSectionArr = {
+            {"//*[@id='mCSB_1_container']/ul/li[1]/a/span", "Home"},
+            {"//*[@id='mCSB_1_container']/ul/li[2]/a/span", "Contact form"},
+            {"//*[@id='mCSB_1_container']/ul/li[3]/a/span", "Service"},
+            {"//*[@id='mCSB_1_container']/ul/li[4]/a/span", "Metals & Colors"},
+            {"//*[@id='mCSB_1_container']/ul/li[5]/a/span", "Elements packs"}
+        };
+
+        for (String[] strings : leftSectionArr) {
+            webElement = webDriver.findElement(By.xpath(strings[0]));
+            String leftSectionText = webElement.getText();
+            assertThat(leftSectionText)
+                .as("Text in the Left Section")
+                .isEqualTo(strings[1]);
+        }
```

Comment:
Same as before
use another locator that returns only required elements and compare collections

Reply:
done